catalystwan/models/monitoring/tunnelhealth.py

```diff
@@ -0,0 +1,80 @@
+from typing import Dict, List, Literal, Optional
+
+from pydantic import BaseModel, Field
+
+HealthValues = Literal["fair", "good", "n/a", "poor"]
+StateValues = Literal["Down", "Up"]
+
+
+# Optional Params for both endpoints.
+class TunnelHealthRequest(BaseModel):
+    last_n_hours: int = Field(default=12, description="Time range for the data in hours.")
+    site: Optional[str] = Field(None, description="Specific site to filter the data.")
```

Comment:
needs default needs to provided as keyword argument to take effect:
```suggestion
    site: Optional[str] = Field(default=None, description="Specific site to filter the data.")
```